git/git.go

```diff
@@ -269,6 +269,11 @@ func Log(args ...string) (*subprocess.BufferedCmd, error) {
 	return gitNoLFSBuffered(logArgs...)
 }
 
+func RefLog(args ...string) (*subprocess.BufferedCmd, error) {
+	reflogArgs := append([]string{"reflog"}, args...)
+	return gitNoLFSBuffered(reflogArgs...)
+}
+
```

Comment:
See my other inline comment on `scanStashed()` for some musings on whether we could dodge the need for this new function by using `git.Log()` to read the stashed refs instead.